crates/kitsune_p2p/kitsune_p2p/src/config.rs

```diff
@@ -70,9 +75,44 @@ fn cnv_bind_to(bind_to: &Option<url2::Url2>) -> TxUrl {
 }
 
 impl KitsuneP2pConfig {
+    #[allow(dead_code)] // because of feature flipping
+    pub(crate) fn is_tx2(&self) -> bool {
+        #[cfg(feature = "tx2")]
+        {
+            #[cfg(feature = "tx5")]
+            {
```

Comment:
This code requires both features tx2 and tx5, is that intended?

Reply:
Yes. We only have to do the match to determine if we're in tx2 mode if tx5 is enabled, otherwise we *know* we're in tx2 mode. This will be different if we ever have a third option